docs/backup_and_restore.rst

```diff
@@ -123,67 +105,216 @@ Restoring
-If you are using the same *Admin Workstation* to do a restore from a previous
-backup, it should already be there because it was placed there by the backup
-command. Otherwise, you should copy the backup archive that you wish to restore to
-``install_files/ansible-base``.
+If you are restoring data onto an existing instance whose onion URLs have not
+changed (for example, for data recovery purposes), see :ref:`Restoring Data on an
+Existing SecureDrop Installation <restore_data>`.
+
+If you are restoring data onto an instance whose onion URLs *have* changed,
+or you are performing a hardware migration, see :ref:`Migrating <migrating>`. 
+
+If your instance has switched from v2 to v3 onion services since your
+most recent backup, see the section 
+:ref:`on mismatched Tor configurations <restore-with-mismatched-tor-config>`, 
+or  `contact Support`_.
 
 .. note:: The backup strategy used for SecureDrop versions prior to 0.3.7
           created encrypted archives with the extension ``.zip.gpg``.
           You can safely remove those files once you've created the ``.tar.gz``
           backup archive described in this guide.
 
-Restoring From a Backup File
-''''''''''''''''''''''''''''
+.. _contact Support: https://securedrop-support.readthedocs.io/en/latest/
 
-.. important:: This documentation applies to a SecureDrop instance using
-               v2 onion services. If your instance uses v3 onion services,
-               you will need to follow additional steps depending on your
-               specific restore scenario.
+.. _restore_data:
 
-To perform a restore, you must already have a backup archive. Provide its
-filename in the following command:
+Restoring Data on an Existing SecureDrop Installation
+''''''''''''''''''''''''''''''''''''''''''''''''''''' 
+
+To restore an existing instance to a previous state, run the command:
 
 .. code:: sh
 
-   ./securedrop-admin restore sd-backup-2017-07-22--01-06-25.tar.gz
+   ./securedrop-admin restore sd-backup-2020-07-22--01-06-25.tar.gz
 
-Make sure to replace ``sd-backup-2017-07-22--01-06-25.tar.gz`` with the filename
+Make sure to replace ``sd-backup-2020-07-22--01-06-25.tar.gz`` with the filename
 for your backup archive. The backup archives are located in
 ``install_files/ansible-base``.
 
-Once the restore is done, the *Application Server* will use the original
-*Source Interface* and *Journalist Interface* Onion URLs. You will need
-to update the corresponding files on the *Admin Workstation*:
+This command restores submissions, source and journalist accounts, and Tor
+configuration (*Source* and *Journalist Interface* onion URLs) to their state at the 
+time of the backup.
 
-* ``app-source-ths``
-* ``app-journalist-aths``
-* ``app-ssh-aths``
+.. _migrating:
 
-Once SSH access to the servers has been established (or if using SSH over
-local network), Onion URLs for the *Source Interface* and *Journalist Interfaces*
-can be fetched using the installer:
```

Comment:
@zenmonkeykstop: I did not use `./securedrop-admin install` to fetch onion URLs _to_ the Admin Workstation. I copied aths/auth-private files into place and ran tailsonfig, so I have provisionally removed this step. If I'm missing something, please let me know.